styles.css

```diff
@@ -1243,8 +1330,736 @@ footer {
   flex-grow: 0;
   margin: 17px 82px;
   list-style: none;
+  justify-content: center;
 }
 
 /* Contact Form section style end here */
 
 /* ------------------------------------------------------------------------------------------------------------------ */
+
+/* ------------------------------------------------------------------------------------------------------------------ */
+
+/* Mobile Media query section starts here */
+
+@media only screen and (max-width: 1180px) {
+  .Illustration {
+    height: 610px;
+    width: 304px;
+  }
+
+  .recent-work {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .feature-post {
+    width: 935px;
+    height: 356px;
+    left: -100px;
+  }
+
+  .Img-Placeholder {
+    background-size: contain;
+    width: 546px;
+    height: 349px;
+  }
+
+  .title-post {
+    width: 327px;
+    height: 48px;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .tag-1 {
+    margin-left: 0;
+    margin-right: 17px;
+  }
+
+  .work-grid {
+    width: 831px;
+  }
+
+  .card1 {
+    width: 900px;
+    height: 448px;
+    left: -101.5px;
+  }
+
+  .card2 {
+    width: 900px;
+    height: 448px;
```

Comment:
- [ Optional ] It is not good to use fixed sized width and height. Because it makes your website unresponsive. You should use percentage instead.